addons/skin.estuary/xml/Variables.xml

```diff
@@ -676,12 +677,109 @@
 		<value condition="VideoPlayer.SubtitlesEnabled">[B]$INFO[VideoPlayer.SubtitlesLanguage][/B]</value>
 		<value>[B]$LOCALIZE[1223][/B]</value>
 	</variable>
-	<variable name="VideoVersionPosterVar">
-		<value condition="!String.IsEmpty(Container(50).ListItem.Art(poster))">$INFO[Container(50).ListItem.Art(poster)]</value>
-		<value condition="!String.IsEmpty(Container(50).ListItem.Art(thumb))">$INFO[Container(50).ListItem.Art(thumb)]</value>
-	</variable>
-	<variable name="VideoVersionFileVar">
-		<value condition="Control.HasFocus(50)">$INFO[Container(50).ListItem.FileNameAndPath]</value>
-		<value>$INFO[ListItem.FileNameAndPath]</value>
+	<variable name="VideoListPosterVar">
+		<value condition="!String.IsEmpty(container(6).listitem.art(poster))">$INFO[container(6).listitem.art(poster)]</value>
+		<value condition="!String.IsEmpty(container(6).listitem.art(thumb))">$INFO[container(6).listitem.art(thumb)]</value>
+		<value condition="!String.IsEmpty(container(50).listitem.art(poster)) + Control.IsVisible(50)">$INFO[container(50).listitem.art(poster)]</value>
+		<value condition="!String.IsEmpty(container(50).listitem.art(thumb))+ Control.IsVisible(50)">$INFO[container(50).listitem.art(thumb)]</value>
```

Comment:
Not sure if it actually matters in terms of function but you missed a space here before the `+`

Reply:
I don't think those Control.IsVisible conditions are required anymore, they are a legacy from the dual lists in the same dialog, so will confirm they can be removed.